Makefile

```diff
@@ -0,0 +1,38 @@
+.DEFAULT_GOAL = help
+
+.PHONY: sh
+sh: ## Get a terminal with Bash.
+	docker-compose exec server bash
+
+.PHONY: logs
+logs: ## Tail the logs, locally.
+	docker-compose logs -f
+
+.PHONY: up
+up: ## Spin up a dev suite of containers.
+	docker-compose up -d
+
+.PHONY: generate
+generate: ## a one time use command needed to setup the db after the creation of the containers
+	docker-compose exec -ti server npx prisma generate --schema server/prisma/schema.prisma 
+
+.PHONY: migrate
+migrate: ## push db changes to the db.
+	docker-compose exec -ti server npx prisma db push --schema server/prisma/schema.prisma
```

Comment:
you technically don't have to run this command on a nosql servers, but mongodb does have db side schema validation so it's better if you do.